wallet-new/server/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -87,11 +88,14 @@ walletBackend WalletBackendParams {..} =
       wsConn <- getWalletWebSockets
       ctx <- V0.walletWebModeContext
       syncWalletsWithGState =<< mapM findKey =<< myRootAddresses
-      startPendingTxsResubmitter
       launchNotifier (V0.convertHandler ctx)
       let app = upgradeApplicationWS wsConn $ serve API.walletAPI (API.walletServer (V0.convertHandler ctx))
       return $ withMiddleware walletRunMode app
 
+-- | A @Plugin@ to resubmit pending transactions.
+resubmitterPlugin :: (HasConfigurations, HasCompileInfo) => Plugin WalletWebMode
+resubmitterPlugin = ([ActionSpec $ \_ _ -> startPendingTxsResubmitter], mempty)
```

Comment:
Woa, I can't believe it works. Previously I couldn't do so because plugins had `RawRealMode` monad or smth like that.